tests/functional/testServers.js

```diff
@@ -152,6 +152,12 @@ describe('Server', function() {
             'VPN is on';
       });
 
+      await vpn.waitForCondition(async () => {
+        return await vpn.getQueryProperty(
+                   queries.screenHome.CONNECTION_TIMER.visible(), 'text') ===
+            '00:00:00';
```

Comment:
Given this is running on 2 Different processes via networking this can be flaky (given we don't know when this will be handled by the client). 
How about we test that it only increments ? i.e 
``` Turn on
 (keep the other checks, i.e name, location  so it's unlikely to be 0)
 let oldTimer = checkTimer()
 Switch servers 
 assert( oldTimer <  checkTimer() ) # We now know it did not reset. 
 
```

Reply:
It's definitely not guaranteed to be instantaneous, but as long as it hits this within the first full second this should be true. If this test ends up being flaky on the server, I'm definitely happy to update it.